csharp/test/Microsoft.ML.OnnxRuntime.EndToEndTests.Capi/CXX_Api_Sample.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+//
+
+#include <assert.h>
+#include <vector>
+#include <onnxruntime_cxx_api.h>
+
+int main(int argc, char* argv[]) {
+  //*************************************************************************
+  // initialize  enviroment...one enviroment per process
+  // enviroment maintains thread pools and other state info
+  Ort::Env env(ORT_LOGGING_LEVEL_WARNING, "test");
+
+  // initialize session options if needed
+  Ort::SessionOptions session_options;
+  session_options.SetThreadPoolSize(1);
+
+  // If onnxruntime.dll is built with CUDA enabled, we can uncomment out this line to use CUDA for this
+  // session (we also need to include cuda_provider_factory.h above which defines it)
+  // #include "cuda_provider_factory.h"
+  // OrtSessionOptionsAppendExecutionProvider_CUDA(session_opsions, 1);
+
+  // Sets graph optimization level
+  // Available levels are 
+  // 0 -> To disable all optimizations
+  // 1 -> To enable basic optimizations (Such as redundant node removals)
+  // 2 -> To enable all optimizations (Includes level 1 + more complex optimizations like node fusions)
+  session_options.SetGraphOptimizationLevel(1);
+
+  //*************************************************************************
+  // create session and load model into memory
+  // using squeezenet version 1.3
+  // URL = https://github.com/onnx/models/tree/master/squeezenet
+  const wchar_t* model_path = L"squeezenet.onnx";
```

Comment:
**wchar_t** seems required for windows, but doesn't work on Linux and Mac (should be char*). 
Adding a comment would help in case someone copy-pastes this into Linux.

Reply:
This is from the existing C_Api_Sample.cpp, it has this same line in there. Should we fix both?

Reply:
Yes, we should probably fix both locations (C_Api as well as the CXX_Api)